sdk/resourcemanager/Azure.ResourceManager/README.md

```diff
@@ -285,7 +281,7 @@ Subscription subscription = await armClient.GetDefaultSubscriptionAsync();
 // Now we get a ResourceGroup collection for that subscription
 ResourceGroupCollection rgCollection = subscription.GetResourceGroups();
 // With GetAllAsync(), we can get a list of the resources in the collection
```

Comment:
Shall we also change the comment here? since we are not actually invoking this function `GetAllAsync`